packages/react-menu/src/components/MenuList/useMenuList.ts

```diff
@@ -38,7 +38,7 @@ export const useMenuList = (
       'aria-labelledby': menuContext.triggerId,
       hasIcons: menuContext.hasIcons,
       hasCheckmarks: menuContext.hasCheckmarks,
-      ...focusAttributes,
+      ...(focusAttributes as any), // eslint-disable-line
```

Comment:
This also shouldn't be necessary, I can still build without it and there is no visible change in `api.md`

Reply:
After I updated to `useTabsterAttrs()` it's no longer required 🚀